_includes/tutorials/error-handling/kstreams/markup/dev/make-exception-handler.adoc

```diff
@@ -65,4 +65,10 @@ Now create the following file at `src/main/java/io/confluent/developer/MaxFailur
 
 You'll add the `StreamsUncaughtExceptionHandler` to your Kafka Streams application in the next step.
 
+There's one more step you'll need to take and that is creating a custom exception class `ProcessorException` that your exception handler will process. Create this simple Java class at `src/main/java/io/confluent/developer/ProcessorException.java`
+
++++++
+<pre class="snippet"><code class="java">{% include_raw tutorials/error-handling/kstreams/code/src/main/java/io/confluent/developer/ProcessorException.java %}</code></pre>
++++++
+
```

Comment:
@bbejeck is the note below:

1. Still accurate?
2. Still needed in the tutorial, or should it be removed?

Reply:
Yes, it's accurate and I think we should leave it as long as the deprecated method is available.